cloudhub/modules/ROOT/pages/app-migration.adoc

```diff
@@ -0,0 +1,142 @@
+
+== Upgrade your CH application to CH2 via Anypoint Platform
+
+. From *Anypoint Platform*, select *Runtime Manager* > *Applications*.
+. Select the CloudHub application you want to upgrade and navigate to *Settings*. 
+. If your application is eligible for the upgrade, the upgrade tooling is available on the *Settings* page: click *Upgrade to CloudHub 2.0*, then click *Start Upgrade*.
++
+image::upgrade-application-window.png[CloudHub to CLoudHub 2.0 application upgrade window.]
++
+The *Upgrade Application to CloudHub 2.0* page opens.
++
+. The *Application Name* field shows the inherited CloudHub application name by default. If you want to use a different name for your CloudHub 2.0 application, change it by editing the field.
+. The *Upgrading From* field shows the name of the CloudHub application the new application is being upgraded from. You can’t edit this field.
+. The *Deployment Target* field shows the preselected private space for the application deployment based on the upgraded CloudHub VPC. You can’t edit this field.
+. For the *Application File*, you can either:
+.. Upload your Application File: Download the application asset from the link provided under *Application File*, make all the necessary changes, and reupload it to deploy your new application to the CloudHub 2.0 private space, or
+.. Import File from Exchange: Import the CloudHub application you want to upgrade.
+. In the *Ingress* tab, specify the *Endpoint* configuration:
+.. Select a *Host* from the dropdown list.
++
+[NOTE] 
+If you're using `anypoint.dns.net` without a TLS context, the Ingress tab doesn’t show it as a possible endpoint.
++
+.. Select a specific *Subdomain* from the dropdown list.
+.. Add a specific *Path*.
+.. Optionally, provide a specific *Path Rewrite*.
+.. Select whether you want to provide internal or external *Access* to your endpoint.
+. Follow the regular steps to xref:cloudhub-2::ch2-deploy.adoc[deploy an application to CloudHub 2.0].
+
+[NOTE]
+The *Host* dropdown list shows only the domains added under the certificates of the underlying VPC Dedicated Load Balancer configuration before the application upgrade started. If you add a new certificate to the underlying VPC Dedicated Load Balancer after the application upgrade has started, you need to manually add it under the *Domain & TLS* tab of your newly created private space for that domain to appear on the *Host* dropdown list.
+
+== Next Steps After Application Upgrade
+
+After the CloudHub 2.0 application has been deployed, you can test and verify it by directing traffic to it and verifying network connectivity and the app behavior. 
+
+If you want to retain the same domains as they move from CloudHub to CloudHub 2.0 to avoid updating your client applications that connect to CloudHub, you can link your applications by applying either of these options:
```

Comment:
```suggestion
To avoid updating your client applications that connect to CloudHub, retain the domains from CloudHub on CloudHub 2.0. To link your applications, apply either of these options:
```